deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml

```diff
@@ -139,7 +142,9 @@ spec:
                 echo 'snapCount=3000000'
                 echo 'leaderServes=yes'
                 echo 'standaloneEnabled=true'
-                echo '4lw.commands.whitelist=stat, ruok, conf, isro, mntr'
+                echo '4lw.commands.whitelist=*'
+                echo 'maxResponseCacheSize=1200'
```

Comment:
Do we have enough RAM for that?

Reply:
according to internal grafana, currently, we use 20-30% of available memory on ZK